its/src/test/cfamilySuite/cfamily.test.ts

```diff
@@ -0,0 +1,90 @@
+/* --------------------------------------------------------------------------------------------
+ * SonarLint for VisualStudio Code
+ * Copyright (C) 2017-2022 SonarSource SA
+ * sonarlint@sonarsource.com
+ * Licensed under the LGPLv3 License. See LICENSE.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import * as assert from 'assert';
+import * as path from 'path';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as vscode from 'vscode';
+
+const fs = require("fs");
+
+const sampleCfamilyFolderLocation = '../../../samples/sample-cfamily/';
+
+suite('CFamily Test Suite', () => {
+  vscode.window.showInformationMessage('Start cfamily tests.');
+
+  test('Extension should be present', () => {
+    assert.ok(vscode.extensions.getExtension('sonarsource.sonarlint-vscode'));
+  });
+
+  test('should report issue on cpp file', async () => {
+    vscode.workspace.getConfiguration().update('sonarlint.pathToCompileCommands', undefined, vscode.ConfigurationTarget.WorkspaceFolder);
```

Comment:
Should be moved to test fixture (beforeEach) or similar: https://mochajs.org/#root-hook-plugins